frontend-react/src/App.tsx

```diff
@@ -104,7 +104,7 @@ const App = () => {
         debounce: 500,
     });
 
-    if (isIE) return <ErrorUnsupportedBrowser />;
+    if (getIsIE()) return <ErrorUnsupportedBrowser />;
```

Comment:
Minor point, but the useragent shouldn't be changing for the app's lifetime so we can hoist this out of the render.